staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -63,19 +65,24 @@ const (
 	// Allow API clients to retrieve resource lists in chunks rather than
 	// all at once.
 	APIListChunking utilfeature.Feature = "APIListChunking"
+
+	KubeAPIServer         = "kube-apiserver"
+	KubeControllerManager = "kube-controller-manager"
 )
 
 func init() {
-	utilfeature.DefaultFeatureGate.Add(defaultKubernetesFeatureGates)
+	if err := utilfeature.DefaultFeatureGate.Add(defaultKubernetesFeatureGates); err != nil {
+		panic(fmt.Sprintf("failed to add features: %v", err))
```

Comment:
is it worth defining a new method to `k8s.io/apiserver/pkg/util` that's something like `AddOrPanic`? we're redefining the functionality a few times here

Reply:
Added `MustAddDefault` for this.